modules/transport_road/LICENSE

```diff
@@ -1,6 +1,8 @@
 MIT License
 
-Copyright (c) 2021, Francesco Sanvito (TU Delft), Bryn Pickering (University of Cambridge), Adrien Mellot (ETH Zürich), Ivan Ruiz Manuel (TU Delft)
+Copyright (c) 2021 Bryn Pickering (ETH Zürich), Tim Tröndle (ETH Zürich / IASS Potsdam)
+Copyright (c) 2024 Adrien Mellot (ETH Zürich)
+Copyright (c) 2024 AUTHORS
```

Comment:
In this case, Adrien is included since he contributed during the sprint.

Reply:
Would remove Tim from the initial copyright